docs/api/README.md

```diff
@@ -959,3 +963,32 @@ Provides some common buffers
 ### `delay(ms, [val])`
 
 Returns a Promise that will resolve after `ms` milliseconds with `val`.
+
+## Cheatsheets
+
+### Blocking / Non-blocking
+
+| Name | Blocking |
+| -- | -- |
+| takeEvery | No |
+| takeLatest | No |
+| throttle | Yes |
+| take | Yes |
+| take(channel) | Sometimes (see API reference) |
+| takem | Yes |
+| put | No |
+| put.resolve (formerly put.sync) | Yes |
+| put(channel, action) | No |
+| call | Yes |
+| apply | Yes |
+| cps | Yes |
+| fork | No |
+| spawn | No |
+| join | Yes |
+| cancel | Yes |
+| select | No |
+| actionChannel | No |
+| flush | Yes |
+| cancelled | Yes |
+| race | Yes |
+| [...effects] | Yes |
```

Comment:
hm, it should have a comment of smth like "depends on the effects, if at least 1 is blocking then it will be blocking, no-blocking otherwise" - could be phrased much better ofc